lib/tasks/hydranorth-solr.rake

```diff
@@ -136,11 +136,38 @@ namespace :hydranorth do
     def index_single(id)
       RakeLogger.info "start reindexing #{id}"
       start = Time.now
-        ActiveFedora::Base.find(id).update_index
+        object = ActiveFedora::Base.find(id)
+        puts object.is_a? GenericFile
```

Comment:
No, the print statement can be removed.

*Weiwei Shi*

Digital Initiative Applications Librarian
University of Alberta Libraries
2-10L Cameron Library
Edmonton, Alberta, Canada T6G 2J8
Phone:(780)492-7802
Fax: (780)248-1209
Email: weiwei.shi@ualberta.ca

On Fri, Sep 2, 2016 at 10:32 AM, pgwillia <notifications@github.com> wrote:

> In lib/tasks/hydranorth-solr.rake
> <https://github.com/ualbertalib/HydraNorth/pull/1248#discussion_r77375165>
> :
>
> > @@ -136,11 +136,38 @@ namespace :hydranorth do
> >      def index_single(id)
> >        RakeLogger.info "start reindexing #{id}"
> >        start = Time.now
> > -        ActiveFedora::Base.find(id).update_index
> > +        object = ActiveFedora::Base.find(id)
> > +        puts object.is_a? GenericFile
>
> Is this necessary?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ualbertalib/HydraNorth/pull/1248/files/245f7836c39128533fa466dd9c9309f52d654908..ab2547fee9bc06cffa5cf8a83bb82c0b42ce4aa5#r77375165>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB8-fnDxvI5dord1AaxnZiMLo6NUr1apks5qmE-qgaJpZM4JwEUT>
> .
>